e2e/kind-config.yaml

```diff
@@ -7,3 +7,5 @@ nodes:
 networking:
   disableDefaultCNI: true # disable kindnet
   podSubnet: 10.42.0.0/16
+  apiServerAddress: 172.18.0.1
```

Comment:
Normally this defaults to `127.0.0.1`, but this is not accessible from the Kilo Pods, so here I'm changing it to the Docker IP address